source/mainnet/smart-contracts/tutorials/sft-minting/build-smart-contract.rst

```diff
@@ -0,0 +1,162 @@
+.. _build-sft-sc:
+
+=================================
+Build and deploy a smart contract
+=================================
+
+Finally, you have everything prepared in order to mint your tokens. Y have created a wallet, exported it, and stored your data on IPFS using Pinata. You have an API Key that will allow you to upload from your system, both your data and metadata are available and pinned already so you don't need to run a local node. You are ready to mint your semi-fungible tokens.
+
+Make sure you are working in the cis2-multi directory, and create a dist folder for your schema and smart contract compiled into wasm. Remember that CIS-2 allows you to mint fungible, non-fungible, and semi-fungible tokens. Concordium’s token standard is applicable to all types of tokens.
+
+.. code-block:: console
+
+    cd cis2-multi
+
+.. code-block:: console
+
+    mkdir -p ../dist/smart-contract-multi
+
+.. code-block:: console
+
+    cargo concordium build --out ../dist/smart-contract-multi/module.wasm --schema-out ../dist/smart-contract-multi/schema.bin
+
+.. image:: images/schema-fns.png
+    :width: 100%
+
+Install required packages
+=========================
+
+You will invoke some functions from your deployed contract using ts-client for minting and transferring NFTs. You can install all the dependent packages with either “yarn” or “npm”. If you don't have the node in your system you should install it first.
+
+.. code-block:: console
+
+    cd node-cli
+
+.. code-block:: console
+
+    yarn install
+
+.. code-block:: console
+
+    yarn add -g ts-node
+
+Deploy contract module
+======================
+
+When you cloned the repository, all necessary functions for node-cli were automatically cloned into the node-cli folder. These functions allow you to interact with the smart contract. Basically, there are 1:1 implementations of all functions in the smart contract. Make sure you have the function below in the cli.ts file.
+
+.. code-block:: console
```

Comment:
Is this section a leftover? Below it says that we will be using concordium-client.

Reply:
Same as previous